beep/structure.py

```diff
@@ -2037,7 +2037,12 @@ def determine_paused(group, paused_threshold=3600):
         for t in date_time_obj
     ]
     date_time_float = pd.Series(date_time_float)
-    return int(date_time_float.diff().max() > paused_threshold)
+    if date_time_float.diff().max() > paused_threshold:
+        paused_secs = date_time_float.diff().max()
+    else:
+        paused_secs = 0
+
```

Comment:
I'm not sure I understand this. If the dataframe has say, 50 events I imagine as a researcher I want one of two things:

1. The sum of: each diff greater than `paused_threshold`
2. The sum of: each diff greater than `paused_threshold` minus paused threshold

(2) considers anything below pause threshold to be some artifact of the machine functioning, some noise if you will.

In our discussions I interpreted you as desiring (1).

This code would return only the maximum paused time that exceeded `paused_threshold`, is that what we want? I'd imagine for a dataframe like:
`[0.0, 3700.0, 7400.0]`

we'd return `7400.0` based on your verbal descriptions of this code but this would return `3700.0` as far as I can tell.

I think assigning `date_time_float.diff().max()` a name might help clear up this confusion for me.